ext/ForwardDiffStaticArraysExt.jl

```diff
@@ -0,0 +1,134 @@
+module ForwardDiffStaticArraysExt
+
+using ForwardDiff
+import DiffResults
+using LinearAlgebra
+import DiffResults: DiffResult, ImmutableDiffResult, MutableDiffResult
```

Comment:
Extensions may only load the package (ForwardDiff) and the weak dependency (StaticArrays), no other packages (see e.g. https://github.com/JuliaStats/LogExpFunctions.jl/pull/63).
```suggestion
using ForwardDiff.DiffResults: DiffResults, DiffResult, ImmutableDiffResult, MutableDiffResult
using ForwardDiff.LinearAlgebra
```